unit/analyses/call_graph.cpp

```diff
@@ -54,6 +54,7 @@ SCENARIO("call_graph",
     // {
     //    A();
     //    B();
+    //    B();
```

Comment:
It seems dangerous to modify the unit test in the same commit since the behaviour of the unit test shouldn't change (since we are using the default value of not collecting callsites). (That is I think adding extra callsites is not a good idea as part of this PR. Extending the unit test to verify the callsites are indeed empty is very diligent :+1:

Reply:
The intent here is to highlight the difference between the multimap representation, which includes duplicate key-value pairs when  there are parallel edges, and the grapht representation which does not.